WPF Frontend

whiterabb17 opened the pull request:
Refactored EDD to be a Dll
EDDCli is the original EDD
Added WPF Frontend for Simple usage of EDD
- Runs each process as a thread, allowing for chained process tasks

whiterabb17 committed:
WPF Frontend

whiterabb17 committed:
EDDCli

whiterabb17 committed:
Update README.md

whiterabb17 committed:
Update Screenshot.png

whiterabb17 committed:
Merge branch 'master' of https://github.com/whiterabb17/EDD

whiterabb17 committed:
Delete Screenshot.png

whiterabb17 committed:
Add files via upload

whiterabb17 committed:
Delete Screenshot.png

whiterabb17 committed:
Add files via upload

whiterabb17 committed:
Update README.md

whiterabb17 committed:
Update MainWindow.xaml.cs

whiterabb17 committed:
Adding projects to Solution

Added all projects to the Solution file.
Removed all prebuilt binaries and objects

ChrisTruncer commented:
Thanks for sending this over. At this point, I don't want to have this completely refactored into a dll because there are uses for it in its current form. With that said, I also can see the obvious value in this addition. Would you possibly want to update this pull request with the latest code updates, and then add it in as a separate folder into this project so people could use it in gui format if they want, or leave use it as is?

whiterabb17 committed:
*GetGroupDomainSid fix (Small Refactor)

Applied the necessary fix to both the Original CLI and the Dll versions.

Refactored EDDCli (the original exe version to EDD)
Refactored EDD (the new dll library to EDDLib)
Moved both GUI & the EDDLib to GUI folder

whiterabb17 committed:
Merge branch 'master' into master
